applications/docs-editor/src/app/Utils/sanitizeUrl.ts

```diff
@@ -3,6 +3,10 @@ export const sanitizeUrl = (url: string): string | undefined => {
     return undefined
   }
 
+  if (url.toLowerCase().startsWith('hxxps')) {
```

Comment:
The check should probably be moved to line 16 and you should use the `parsedUrl.protocol` for the check. Also check `hxxps` and `hxxp`.